repo_e/sum_words_string.py

```diff
@@ -0,0 +1,12 @@
+def sum_word_in_string(string):
+    counts = dict()
+    str_split = str.split(string)
+    for word in str_split:
```

Comment:
same here, please try to name it better, str_split could be named `words` instead